src/server/routes/entity/publisher.js

```diff
@@ -58,6 +58,16 @@ function _setPublisherTitle(res) {
 	);
 }
 
+/**
+ * @param {object} a The first object.
+ * @param {object} b The second object.
+ * @returns {number} The sum of the two numbers.
+ */
+function SortByDate(a, b) {
```

Comment:
Please use camelCase for function names - make sure you're running "npm run lint" prior to comitting.

Reply:
@LordSputnik isn't it already in CamelCase?

Reply:
The camelCase standard used in JavaScript is to have an initial lower case letter instead of an upper case letter.